src/main/java/bio/terra/workspace/service/workspace/model/WorkspaceRequest.java

```diff
@@ -0,0 +1,70 @@
+package bio.terra.workspace.service.workspace.model;
+
+import bio.terra.workspace.service.spendprofile.SpendProfileId;
+import com.google.auto.value.AutoValue;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * Internal representation of a request to create a workspace.
+ *
+ * <p>While this object is nearly identical to the {@code Workspace} object, there is a semantic
+ * difference: a {@code Workspace} is a fully formed workspace, while a {@code WorkspaceRequest}
+ * only contains fields specified by clients. These objects may diverge in the future if properties
+ * of a Workspace are generated internally.
```

Comment:
the future is now with operation id. I think noting the semantic difference is enough. This is also a comment that's likely to not get updated to be correct when the future arrives.